plugin/dnstap/setup_test.go

```diff
@@ -44,14 +45,16 @@ func TestConfig(t *testing.T) {
                 identity NAME2
                 version VER2
                 extra EXTRA2
+				message_types CLIENT_RESPONSE CLIENT_QUERY
               }`, false, []results{
-			{"dnstap.sock", true, "unix", []byte("NAME"), []byte("VER"), "EXTRA"},
-			{"127.0.0.1:6000", false, "tcp", []byte("NAME2"), []byte("VER2"), "EXTRA2"},
+			{"dnstap.sock", true, "unix", []byte("NAME"), []byte("VER"), "EXTRA", defaultEnabledMessageTypes},
+			{"127.0.0.1:6000", false, "tcp", []byte("NAME2"), []byte("VER2"), "EXTRA2", 0b1100000},
```

Comment:
you can still add a comment here to designate which message types are encoded in bit mask